include/dali/core/event_pool.h

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Why  not name file as cuda_event_pool to keep it consistent with the class name?